src/main/java/org/reactome/orthoinference/SkipInstanceChecker.java

```diff
@@ -23,14 +17,17 @@
 	private static final Logger logger = LogManager.getLogger();
 	private static MySQLAdaptor dba;
 	private static Set<String> skipList = new HashSet<>();
+	private static final long HIV_INFECTION_DBID = 162906L;
+	private static final long INFLUENCE_INFECTION_DBID = 168255L;
+	private static final long AMYLOID_FIBER_FORMATION_DBID = 977225L;
 
 	// Skiplist was traditionally provided in a file, but since it's currently just 3 instances, I've just hard-coded them here.
 	public static void getSkipList(String pathToSkipList) throws Exception
 	{
-		String[] pathwayIdsToSkip = {"162906","168254","977225"};
-		for (String pathwayId : pathwayIdsToSkip)
+		List<Long> pathwayIdsToSkip = Arrays.asList(HIV_INFECTION_DBID, INFLUENCE_INFECTION_DBID, AMYLOID_FIBER_FORMATION_DBID);
+		for (Long pathwayId : pathwayIdsToSkip)
```

Comment:
Could the type of `pathwayId` be `long`?  Using primitives wherever possible helps avoid object value vs. reference equality issues.